js/co-authors-plus.js

```diff
@@ -1,533 +1,504 @@
-			// Including existing authors on the AJAX suggest link
-			// allows us to filter them out of the search request
-			var existing_authors = jQuery('input[name="coauthors[]"]').map(function(){return jQuery(this).val();}).get();
-			settings.url = settings.url.split('&existing_authors')[0];
-			settings.url += '&existing_authors=' + existing_authors.join(',');
-			show_loading();
-		}
-	});
-	// Hide laoding cursor when autocomplete ajax requests are finished
-	jQuery(document).ajaxComplete(function(e, xhr, settings) {
-		if( settings.url.indexOf(coAuthorsPlus_ajax_suggest_link) != -1 )
-			hide_loading();
-	});
-
-	if ( 'post-php' == adminpage || 'post-new-php' == adminpage ) {
-		var $post_coauthor_logins = jQuery('input[name="coauthors[]"]');
-		var $post_coauthor_names = jQuery('input[name="coauthorsinput[]"]');
-		var $post_coauthor_emails = jQuery('input[name="coauthorsemails[]"]');
-		var $post_coauthor_nicenames = jQuery('input[name="coauthorsnicenames[]"]');
-
-		var post_coauthors = [];
-
-		for(var i = 0; i < $post_coauthor_logins.length; i++) {
-			post_coauthors.push({
-				login: $post_coauthor_logins[i].value,
-				name: $post_coauthor_names[i].value,
-				email: $post_coauthor_emails[i].value,
-				nicename: $post_coauthor_nicenames[i].value
-			});
-		}
-
-		// Remove the read-only coauthors so we don't get craziness
-		jQuery('#coauthors-readonly').remove();
-		coauthors_initialize(post_coauthors);
-	}
-	else if ( 'edit-php' == adminpage ) {
-
-		var wpInlineEdit = inlineEditPost.edit;
-
-		inlineEditPost.edit = function( id ) {
-
-			wpInlineEdit.apply( this, arguments )
-
-			// get the post ID
-			var postId = 0
-			if ( typeof( id ) == 'object' )
-				postId = parseInt( this.getId( id ) )
-
-			if ( postId > 0 ) {
-
-				var $postRow = jQuery( '#post-' + postId )
-
-				// Move the element to the appropriate position in the view
-				// JS hack for core bug: https://core.trac.wordpress.org/ticket/26982
-				jQuery('.quick-edit-row .inline-edit-col-left .inline-edit-col').find('.inline-edit-coauthors').remove() // remove any previously added elements
-				var el = jQuery('.inline-edit-group.inline-edit-coauthors', '#edit-' + postId );
-				el.detach().appendTo('.quick-edit-row .inline-edit-col-left .inline-edit-col').show();
-
-				// initialize coauthors
-				var post_coauthors = jQuery.map(jQuery('.column-coauthors a', $postRow), function(el) {
-					return {
-						login: jQuery(el).data('user_login'),
-						name: jQuery(el).data('display_name'),
-						email: jQuery(el).data('user_email'),
-						nicename: jQuery(el).data('user_nicename')
-					}
-				})
-				coauthors_initialize(post_coauthors);
-
-			}
-		}
-	}
-
-});
-
-if( typeof(console) === 'undefined' ) {
-	var console = {}
-	console.log = console.error = function() {};
-}
\ No newline at end of file
+jQuery( document ).ready(function () {
+
+	/*
+     * Click handler for the delete button
+     * @param event
```

Comment:
There's some weirdness happening with the indentation for docblocks in this file.